luma_core/src/sysconf.rs

```diff
@@ -0,0 +1,184 @@
+//! ``sysconf`` module of ``luma_core``.
+//!
+//! Provides access to the Wii system settings, as set by the system menu.
+
+use crate::ios;
+use alloc::collections::BTreeMap;
+use alloc::string::{String, ToString};
+use alloc::vec::Vec;
+use core::fmt;
+
+#[derive(Debug, Clone)]
+enum Item {
+    BigArray(Vec<u8>),
+    SmallArray(Vec<u8>),
```

Comment:
I don't think allocations are needed here, maybe use `'a [u8]` for both instead.